lua/lvim/core/telescope.lua

```diff
@@ -7,21 +7,22 @@ local M = {}
 ---| "center"   # retain the default telescope theme
 
 function M.config()
-  -- Define this minimal config so that it's available if telescope is not yet available.
+  local lazy_action = function(name)
```

Comment:
is this necessary? I thought it would've been covered by one of the utility functions

Reply:
it's necessary unless I put `actions.smart_send_to_qflist + actions.open_qflist` in a function which I guess is a more readable and easier approach